testdata/sqllogictests_clickhouse/datatypes.slt

```diff
@@ -0,0 +1,47 @@
+# Test if the datasource supports the different datatypes (and NULLs).
+
+# Create an external table that connects to the datatypes table.
+statement ok
+CREATE EXTERNAL TABLE datatypes
+	FROM clickhouse
+	OPTIONS (
+		connection_string = '${CLICKHOUSE_CONN_STRING}',
+		table = 'datatypes'
+	);
+
+# Check if we can fetch contents of the datatype table.
+query ITIIIIIIIIRRTTTTTTT rowsort
+SELECT * FROM datatypes;
+----
+1  t  1  12  1234  12345678  -1  -12  -1234  -12345678  1.25  -34.625  abc  def  1999-09-30  1999-09-30 16:32:34  1999-09-30 16:32:34.123456+00  1999-09-30 11:02:34  1999-09-30 16:32:34.123456+00
+2  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL
+
+# Check the actual types of columns.
+
+skipif glaredb_rpc
+
+query TTTTTTTTTTTTTTTTTTT
+SELECT
+	arrow_typeof(c1),
+	arrow_typeof(c2),
+	arrow_typeof(c3),
+	arrow_typeof(c4),
+	arrow_typeof(c5),
+	arrow_typeof(c6),
+	arrow_typeof(c7),
+	arrow_typeof(c8),
+	arrow_typeof(c9),
+	arrow_typeof(c10),
+	arrow_typeof(c11),
+	arrow_typeof(c12),
+	arrow_typeof(c13),
+	arrow_typeof(c14),
+	-- arrow_typeof(c15),
```

Comment:
why is this one skipped?

Reply:
There isn't a column called `c15` due to lack of support in the clickhouse library.